src/client/firebase/auth.js

```diff
@@ -1,16 +1,26 @@
 function handleAuthErrors({ code, message }) {
```

Comment:
There is already a seperate pr for this. Also, try to limit the pr to the issue only. This makes debugging easier

Reply:
Here #16

Reply:
Now I see, my mistake is that I added eslint exceptions because I was having some problems, but my PR is for Frontend: Dummy Pages. I will try to revert the commit with the eslint exceptions.